### PR TITLE
Update version in POMs

### PR DESCRIPTION
**Pull request by KatrukOV**
**A Brief Overview**
Updated versions in pom: spring, closure.compiler, jacoco-maven-plugin, groovy; remove commons-logging

Link to QA issue
[QA-4983](https://github.com/BroadleafCommerce/QA/issues/4983)

### CODE REVIEW

**Review by agustinlacombadev — APPROVED** [review]

File: integration/pom.xml [hunk 1]
```diff
@@ -202,6 +202,16 @@
             <artifactId>spring-boot-test</artifactId>
             <version>${spring.boot.version}</version>
             <scope>test</scope>
+            <exclusions>
```
Comment (oleksiimiroshnyk) [inline integration/pom.xml:205]: looks like this changes are from 6.3 not 6.2 ?

**Review by oleksiimiroshnyk — APPROVED** [review]

**Comment by riteshadhikari17** [comment]
~~DO NOT MERGE!! These exclusions are sensitive things. I'll have test this with plugins like 'dependency-check' and 'enforcer' as well as verify versions and make sure it didn't break anything.~~
Also there are new vulnerabilities being reported so will probably have to update other libs too.

EDIT: Updated

File: integration/pom.xml (hunk 1 above)
Comment (riteshadhikari17) [reply integration/pom.xml:205]: @KatrukOV Is this relevant? It seems ok in my smoke test and doesn't seem to cause issues. Have you tested these changes with Refsite? I am ok with merging this if alex is ok with this.

Comment (KatrukOV) [reply integration/pom.xml:205]: Yes, this is relevant, we checked it.

Comment (oleksiimiroshnyk) [reply integration/pom.xml:205]: I'm ok with this change

**Review by riteshadhikari17 — APPROVED** [review]

**Pull request merged**